go/chat/sync.go

```diff
@@ -491,14 +521,18 @@ func (s *Syncer) sync(ctx context.Context, cli chat1.RemoteInterface, uid gregor
 				}
 				queuedConvs++
 			} else {
-				// If we are a limited data connection only queue selected
-				// convs up to maxConvs
-				if state.IsLimited() && (queuedConvs >= maxConvs || !s.shouldUnboxSyncConv(conv)) {
+				// Only do this for chat convs
+				if conv.GetTopicType() != chat1.TopicType_CHAT {
```

Comment:
why not move this into `shouldUnboxSyncConv`?

Reply:
Need KBFS convs...

Reply:
Maybe we can ignore them on mobile.